src/logic/safe/hooks/useOwnerSafes.ts

```diff
@@ -8,39 +8,54 @@ import useCachedState from 'src/utils/storage/useCachedState'
 
 type OwnedSafesCache = Record<string, Record<string, string[]>>
 
+type UseOwnerSafesType = {
+  isLoading: boolean
+  ownerSafes: Record<string, string[]>
+}
+
 const storageKey = 'ownedSafes'
 
-const useOwnerSafes = (): Record<string, string[]> => {
+const useOwnerSafes = (): UseOwnerSafesType => {
   const connectedWalletAddress = useSelector(userAccountSelector)
   const chainId = useSelector(currentChainId)
   const [cache = {}, setCache] = useCachedState<OwnedSafesCache>(storageKey)
   const ownerSafes = cache[connectedWalletAddress] || {}
+  const [isLoading, setIsLoading] = useState<boolean>(true)
```

Comment:
I would suggest now initialising this with a value so that we can determine if the state was set somewhere.

Reply:
Can you explain? Maybe add a suggestion

Reply:
Sorry, without* a value: `useState<boolean>()`.

Reply:
Ahh you mean "not" ?

Reply:
Yes 🙈